pkg/sql/planner.go

```diff
@@ -651,6 +651,11 @@ const (
 	KVStringOptRequireValue   KVStringOptValidate = `value`
 )
 
+// KVStringCaseInsensitiveOpts options which supports case Insensitive value
+var KVStringCaseInsensitiveOpts = map[string]struct{}{
```

Comment:
Thanks for the PR!

We should probably move this var into a changefeed package since we're only dealing with the supported options at the changefeed level. Perhaps [`changefeedbase/options.go`](https://github.com/cockroachdb/cockroach/blob/master/pkg/ccl/changefeedccl/changefeedbase/options.go) and call it something like `CaseInsensitiveOpts`?